### PR TITLE
Packaging Intel 471 Solution

### PR DESCRIPTION
**Pull request by mmolenda**
Change(s):
   - Adding mainTemplate.json and related package files

   Reason for Change(s):
   - Package for the first release

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Hi @mmolenda, please repackage this solution with https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Create-Azure-Sentinel-Solution/V2 with version 2.0.0.
Also include Solutions/Intel471/Data/Solution_Intel471.json file.

Can you please fix the failing playbook validation? Thanks

**Comment by mmolenda** [comment]
Hi @v-rbajaj, thanks for the review. I added Solution_Intel471.json file and I fixed the validation issues.
The package was already generated using V2 tool.

**Comment by v-rbajaj** [comment]
> Hi @v-rbajaj, thanks for the review. I added Solution_Intel471.json file and I fixed the validation issues. The package was already generated using V2 tool.

Hi @mmolenda, the zip version is 1.0.0, please package this with 2.0.0 and remove 1.0.0 zip.

**Comment by mmolenda** [comment]
Hi @v-rbajaj is there anything I need to complete in order to get it merged?

File: Solutions/Intel471/Playbooks/Intel471-ImportMalwareIntelligenceToSentinel/README.md [hunk 1]
```diff
@@ -46,12 +46,12 @@ for building the Logic App along with the necessary connections. The ARM builds
     * **Action**: The action to apply if the indicator is matched from within the targetProduct security tool. Allowed values: `unknown`, `allow`, `block`, `alert`
     * **Look Back Days**: How many days of history should be pulled on the first run. Leave 0 to start from the current time
 
-    [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FIntel471%2FPlaybooks%2FIntel471-ImportMalwareIntelligenceToSentinel%2Fazuredeploy.json) [![Deploy to Azure](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FIntel471%2FPlaybooks%2FIntel471-ImportMalwareIntelligenceToSentinel%2Fazuredeploy.json)
+    [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FIntel471%2FPlaybooks%2FIntel471-ImportMalwareIntelligenceToSentinel%2Fazuredeploy.json)
```
Comment (manishkumar1991) [inline Solutions/Intel471/Playbooks/Intel471-ImportMalwareIntelligenceToSentinel/README.md:50]: "Deploy to azure gov " button is missing in readme file .
Please take reference from below link :
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-BlockIP-URL/readme.md

**Comment by manishkumar1991** [comment]
@mmolenda  , It would be highly appreciated, if you can provide us the images of working flow of the playbooks, 
By creating separate folder of images in location where arm template file of playbook is provided.

**Comment by mmolenda** [comment]
@manishkumar1991 after adding screenshot and the button in Readme I can see two failed jobs with no further explanation. Can you take a look please?

**Comment by v-rbajaj** [comment]
> @manishkumar1991 after adding screenshot and the button in Readme I can see two failed jobs with no further explanation. Can you take a look please?

Hi @mmolenda, we are looking into this validation fix.

**Comment by v-rbajaj** [comment]
@mmolenda, can you please update the branch from master and push the changes?

**Review by v-rbajaj — APPROVED** [review]
Looks good

**Comment by manishkumar1991** [comment]
playbook looks good.

**Review by manishkumar1991 — APPROVED** [review]
Playbook , approved

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**